financialaid/management/commands/generate_exchange_rates_test.py

```diff
@@ -0,0 +1,47 @@
+"""
+Test for management command generating exchange rates
+"""
+from mock import patch
+
+from financialaid.constants import CURRENCY_EXCHANGE_RATE_API_REQUEST_URL
+from financialaid.management.commands import generate_exchange_rates
+from financialaid.models import CurrencyExchangeRate
+from search.base import ESTestCase
+
+
+@patch('financialaid.tasks.requests.get')
+class GenerateExchangeRatesTest(ESTestCase):
```

Comment:
Ah, OK. In that case, we can still inherit from `ESTestCase`. (Although perhaps that class should be renamed to something more generic, if we want *all* tests to inherit from it...)